README.md

```diff
@@ -1,17 +1,31 @@
 # Memo Index
 
+### Install deps
+```bash
+dnf install -y golang,bzr
+```
+
 ### Checkout repo
 ```bash
 git clone git@github.com:memocash/index.git
 cd index
-go build
+export GOVCS='*:bzr|git'; go build
```

Comment:
This is not required for standard Golang installation.

Reply:
It seems like it is needed on the latest Fedora with `dnf install golang breezy`, however I'm getting an error:

`
[blake@fedora index]$ GOVCS=*:all go build
go: github.com/tyler-smith/go-bip32@v1.0.0 requires
	launchpad.net/gocheck@v0.0.0-20140225173054-000000000087: invalid version: bzr pull --overwrite-tags in /home/blake/go/pkg/mod/cache/vcs/f46ce2ae80d31f9b0a29099baa203e3b6d269dace4e5357a2cf74bd109e13339: exit status 3:
	brz: ERROR: Not a branch: "/home/blake/go/pkg/mod/cache/vcs/f46ce2ae80d31f9b0a29099baa203e3b6d269dace4e5357a2cf74bd109e13339/.bzr/branch/": location is a repository.
go: github.com/tyler-smith/go-bip32@v1.0.0 requires
	launchpad.net/gocheck@v0.0.0-20140225173054-000000000087: invalid version: bzr pull --overwrite-tags in /home/blake/go/pkg/mod/cache/vcs/f46ce2ae80d31f9b0a29099baa203e3b6d269dace4e5357a2cf74bd109e13339: exit status 3:
	brz: ERROR: Not a branch: "/home/blake/go/pkg/mod/cache/vcs/f46ce2ae80d31f9b0a29099baa203e3b6d269dace4e5357a2cf74bd109e13339/.bzr/branch/": location is a repository.
`

Reply:
I mentioned this in another comment. I would recommend following Golang installation instructions. It seems like the issue you guys are running into is with the Fedora package manager.

https://go.dev/doc/install